src/test/java/com/bloxbean/cardano/client/crypto/bip32/HdKeyGeneratorTest.java

```diff
@@ -49,4 +50,18 @@ void testGetRootKeyPairFromEntropy_whenAndroid() throws MnemonicException.Mnemon
         assertThat(rootKey).isEqualTo("root_xsk1hp9an83kfma0ufdaeqft6xv0snf4ek9uqemk5chp3p25el8w0fglszmgkq9qxvguj33fnulms4qfnx9jawhde4ng9qzg3zzg5u8r2af4jxpe8nfjulrzey7p8ttnt5yn53exsawm6wkmtqm989ehwtr0kc244zfn");
 
     }
+
+    @Test
+    void testPubKeyFromParentPubKey() {
```

Comment:
Should we add some more tests? Please check if any test vector available for bip32-ed25519. Otherwise, you can ignore that for now. I will check after merge.